File: app/src/main/java/wangdaye/com/geometricweather/daily/adapter/model/Value.java
package wangdaye.com.geometricweather.daily.adapter.model;

import wangdaye.com.geometricweather.daily.adapter.DailyWeatherAdapter;

public class Value implements DailyWeatherAdapter.ViewModel {

    private String title;
    private String value;

    public Value(String title, String value) {
        this.title = title;
        this.value = value;
    }

    public String getTitle() {
        return title;
    }

    public void setTitle(String title) {
        this.title = title;
    }

    public String getValue() {
        return value;
    }

    public void setValue(String value) {
        this.value = value;
    }

    public static boolean isCode(int code) {
        return code == 3;
    }

    @Override
    public int getCode() {
        return 3;
    }
}

File: app/src/main/java/org/breezyweather/daily/adapter/model/Value.kt
package org.breezyweather.daily.adapter.model

import org.breezyweather.daily.adapter.DailyWeatherAdapter

class Value(
    val title: String,
    val value: String
) : DailyWeatherAdapter.ViewModel {
    override val code = 3

    companion object {
        fun isCode(code: Int) = (code == 3)
    }
}
